src/views/OnlineLeaguePlayerLanding/OnlineLeaguePlayerLanding.test.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react";
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+import { act, render, screen } from "@testing-library/react";
+import * as ogs_hooks from "hooks";
+
+import { OgsHelpProvider } from "OgsHelpProvider";
+
+import * as requests from "requests";
+import { OnlineLeaguePlayerLanding } from "./OnlineLeaguePlayerLanding";
+
+// This thing has a TabCompleteInput in it that doesn't work under jest
+import { EmbeddedChatCard } from "../../../src/components/Chat";
+
+afterEach(() => {
+    (EmbeddedChatCard as jest.Mock).mockClear();
+});
+
+jest.mock("../../../src/components/Chat", () => {
+    return {
+        EmbeddedChatCard: jest.fn(() => {
+            return <div className="EmbeddedChatCardMock" />;
+        }),
+    };
+});
+
+const TEST_USER = {
+    username: "testuser",
+    anonymous: false,
+    id: 0,
+    registration_date: "",
+    ratings: undefined,
+    country: "",
+    professional: false,
+    ranking: 0,
+    provisional: 0,
+    can_create_tournaments: false,
+    is_moderator: false,
+    is_superuser: false,
+    is_tournament_moderator: false,
+    supporter: false,
+    supporter_level: 0,
+    tournament_admin: false,
+    ui_class: "",
+    icon: "",
+    email: "",
+    email_validated: "",
+    is_announcer: false,
+} as const;
+
+/*
+const UNSTARTED_MATCH = {
+    id: 1,
+    name: "Test Match 2",
+    league: "TL",
+    player_key: "blackmatchid",
+    side: "black",
+    started: false,
+    game: 0, // pk
+    black_ready: false,
+    white_ready: false,
+} as const;
+*/
+
+jest.mock("requests");
+
+describe("COOL Player landing tests", () => {
+    test("logged out player arrival", async () => {
+        const user = { ...TEST_USER, anonymous: true };
+        jest.spyOn(ogs_hooks, "useUser").mockReturnValue(user);
+        // data.set("user", user);
```

Comment:
Note that I left this in because I was bumping into a place where data.set is used rather than useUser *in a place where a hook can't be used*.    BUT at some point I changed something that sidestepped that ... I was thinking "hmmm, I can't see why that went away, I'll be needing this back" :)